### PR TITLE
DOGE-54: convenience method to get full function logs

### PR DESCRIPTION
**Pull request by ddonukis**
## Description
Added a `to_text` method on the `FunctionCallLog` class that formats log entries into a new line delimited string.

## Checklist:
- [x] Tests added/updated.
- [x] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by erlendvollset — APPROVED** [review]
LGTM! Please update changelog and bump version aswell before mergin.

File: cognite/client/data_classes/functions.py [hunk 1]
```diff
@@ -376,10 +377,28 @@ def __init__(
...
+
     _RESOURCE = FunctionCallLogEntry
 
+    def to_text(self, with_timestamps: bool = False) -> str:
+        """Return a new-line delimeted string of the log entry messages, optionally with entry timestamps.
+
+        Args:
+            with_timestamps (bool): Whether to include entrry timestamps in the output. Defaults to False.
+        Returns:
+            str: new-line delimeted log entries.
```
Comment (haakonvt) [inline cognite/client/data_classes/functions.py:398]:
```suggestion
        """Return a new-line delimited string of the log entry messages, optionally with entry timestamps.

        Args:
            with_timestamps (bool): Whether to include entry timestamps in the output. Defaults to False.
        Returns:
            str: new-line delimited log entries.
```

Comment (haakonvt) [inline cognite/client/data_classes/functions.py:398]:
```suggestion
        """Return a new-line delimited string of the log entry messages, optionally with entry timestamps.

        Args:
            with_timestamps (bool): Whether to include entry timestamps in the output. Defaults to False.
        Returns:
            str: new-line delimited log entries.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.28.6] - 2023-10-04
+### Added
+- Convenience method `to_text` on the `FunctionCallLog` class that formats log entries into a new-line delimited string.
```
Comment (haakonvt) [inline CHANGELOG.md:22]: I think a better formulation would be to state how this helps the user nicely printing the output logs of a function call 😄

File: cognite/client/data_classes/functions.py [hunk 3]
```diff
@@ -376,10 +377,28 @@ def __init__(
         self.message = message
         self._cognite_client = cast("CogniteClient", cognite_client)
 
+    def _format(self, with_timestamps: bool = False) -> str:
+        ts = ""
+        if with_timestamps and self.timestamp is not None:
+            ts = f"[{ms_to_datetime(self.timestamp).isoformat()}] "
```
[the remark this answers is not in the document]
Comment (haakonvt) [reply cognite/client/data_classes/functions.py]: Not a problem:

IMG_0421

**Pull request merged**